test/opt/pass_merge_return_test.cpp

```diff
@@ -489,6 +489,9 @@ TEST_F(MergeReturnPassTest, SplitBlockUsedInPhi) {
   SinglePassRunAndMatch<MergeReturnPass>(before, false);
 }
 
+// TODO(#1861): Reenable these test when the breaks from selection constructs
+// are reenabled.
+/*
```

Comment:
The proper way to disable a test is to prefix the test name with `DISABLED_`.  Then it is still enumerated and counted but not executed.

https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#temporarily-disabling-tests